### PR TITLE
NO-TICKET: Remove DeployBuffer from joiner reactor.

### PR DESCRIPTION
**Pull request by goral09**
This PR removes `DeployBuffer` from the `joiner` reactor. It wasn't needed, I must have copy-pasted too much when debugging syncing issues.

Updates `run-dev.sh` script as well to work with the new `*_timestamp` formats.

### CODE REVIEW

File: run-dev.sh [hunk 1]
```diff
@@ -9,17 +9,17 @@ CHAINSPEC=$(mktemp -t chainspec_XXXXXXXX --suffix .toml)
 TRUSTED_HASH="${TRUSTED_HASH:-}"
 
 # Generate a genesis timestamp 30 seconds into the future, unless explicity given a different one.
-OFFSET="${OFFSET:-30}"
-TIMESTAMP=${GENESIS_TIMESTAMP:-$(date '+%s000' -d "+$OFFSET sec")}
+TIMESTAMP=$(python3 -c 'from datetime import datetime, timedelta; print((datetime.utcnow() + timedelta(seconds=20)).isoformat("T") + "Z")')
```
Comment (afck) [inline run-dev.sh:12]: Can't `date` do that as well?

**Review by afck — APPROVED** [review]

File: run-dev.sh (hunk 1 above)
Comment (goral09) [reply run-dev.sh:12]: I don't know really. I copied this from the `nctl` code that Fraser wrote.

**Review by EdHastingsCasperLabs — APPROVED** [review]

**Pull request merged**